.github/workflows/main.yml

```diff
@@ -79,13 +79,9 @@ jobs:
               PRIMARY: "substratevm"
           - env:
               JDK_VERSION: "21"
-              GATE_TAGS: "build,test,helloworld"
```

Comment:
I guess it makes sense to replace that with something else. This is what the internal CI is running:
https://github.com/oracle/graal/blob/master/substratevm/ci/ci.jsonnet#L115
So maybe run `native_unittests` here...

Reply:
Interesting, I didn't notice we were not running `native_unittests`... Adding it, thanks